doc/source/serve/production.md

```diff
@@ -1,66 +1,63 @@
 (serve-in-production)=
 
-# Putting Ray Serve Deployment Graphs in Production
+# Production Guide
+
+The recommended way to run Ray Serve in production is on Kubernetes using the KubeRay [RayService] custom resource because it automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
+If you're not running on Kubernetes, you can also run Ray Serve on a single node or on a multi-node cluster (likely managed using the [Ray Cluster Launcher](cluster-cloud)) using the Serve CLI.
 
 This section should help you:
 
-- develop and test your Serve deployment graph locally
-- understand the Serve config file format
-- deploy, inspect, and update your Serve application in production
+- understand the Serve config file format.
+- understand how to deploy Ray Serve using the REST API / CLI directly.
+- understand how to deploy Ray Serve on Kubernetes using the [RayService] custom resource.
```

Comment:
we have been using lower case across the pages because the bullets are a continuation of the sentence above. If we change it we'll need to change all of the others. I'm not really opinionated either way but prefer to keep it uniform.